NEventstore.Persistence.DocumentDB/Constraints/Logical_Key.js

```diff
@@ -0,0 +1,51 @@
+﻿function LogicalKey_Constraint() {
```

Comment:
An extra thought, we need to ensure that this trigger approach is thread safe too?